AppointmentReminders.Web/Content/bootstrap-datetimepicker.css

```diff
@@ -0,0 +1,372 @@
+/*!
```

Comment:
I'll review the unused dependencies, and rid of them. To use a CDN or a local library is almost like a trivial decision, so I'll keep using a local copy, this shouldn't be a big deal, right?